examples/connext_dds/dynamic_data_request_reply/cpp/USER_QOS_PROFILES.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0"?>
+
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="http://community.rti.com/schema/6.0.1/rti_dds_qos_profiles.xsd">
+
+    <qos_library name="RequestReplyExampleProfiles">
+    
+        <!-- Default QoS:
+             
+             This profile contains the QoS that Requesters and Repliers 
+             would use by default. We can use it as a base profile to inherit
+             from and override some parameters 
+         -->
+        <qos_profile name="default">
```

Comment:
it's recommended that this profile inherits from another (base_name ="Library::name")

Furthermore, we can use the QoS Composition pattern (this only works from 6.0.0 and on): https://community.rti.com/best-practices/qos-profile-inheritance-and-composition-guidance

And the builtin QoS Snippetes (6.0.1+)